interceptors/ratelimit/ratelimit.go

```diff
@@ -9,17 +9,17 @@ import (
 )
 
 // Limiter defines the interface to perform request rate limiting.
-// If Limit function return true, the request will be rejected.
+// If Limit function returns an error, the request will be rejected with XXX status code and the provided error as the response.
```

Comment:
XXX? That was a placeholder ;p

Reply:
it sounded to me a bit suspicious actually :P ok removed :raccoon: